target/linux/ar71xx/image/generic.mk

```diff
@@ -196,6 +209,45 @@ define Device/mr16
 endef
 TARGET_DEVICES += mr16
 
+define Device/mr1750
+  DEVICE_TITLE := OpenMesh MR1750
+  DEVICE_PACKAGES := om-watchdog kmod-ath10k ath10k-firmware-qca988x
+  BOARDNAME = MR1750
+  FILESYSTEMS := squashfs
+  BLOCKSIZE := 4 8 16 64 128 256
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image board=MR1750 rootfs=$$$$@
```

Comment:
No, this was simply a test. I wanted to have more than the default filesystems selected to test what happens when i remove "FILESYSTEMS" and/or the "/squashfs".

The default settings of LEDE work perfectly fine. But non-default settings of LEDE should not cause a build error due to the OpenMesh device image code or create invalid images.

You can use it therefore as an example to see why I prefer to have the FILESYSTEMS line and the /squashfs suffix.